openslides_backend/migrations/migrations/0062_unset_presence_of_removed_users.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+from datastore.migrations import BaseModelMigration
+from datastore.writer.core import BaseRequestEvent, RequestUpdateEvent
+
+from openslides_backend.shared.patterns import fqid_from_collection_and_id
+
+
+class Migration(BaseModelMigration):
+    """
+    This migration adds the user_id "-1" to all existing action_workers.
+    This is the number usually used for calls using the internal route.
```

Comment:
user_id `-1` is not an Id and will lead to errors. Why use such an id?

Reply:
Wrote the actual description.